### ISSUE TITLE
baresip fails to compile on archlinux on raspi

### ISSUE THREAD

**Issue by lisleadam** [alfredh/baresip#612]
I'm running archlinux (ArchLinuxARM-rpi-2) on raspi3b+ and have been trying to make baresip 0.6.0, but it fails with the following ld error (excerpt);

`/usr/bin/ld: build-arm/modules/omx/module.o:/home/USERNAME/baresip/baresip-0.6.0/modules/omx/module.c:126: multiple definition of `exports'; build-arm/modules/omx/module.o:/home/USERNAME/baresip/baresip-0.6.0/modules/omx/module.c:126: first defined here
collect2: error: ld returned 1 exit status
make: *** [mk/mod.mk:36: omx.so] Error 1
`
In fact there are several "multiple definition of 'exports'" reports other than this.
re, rem and openssl are built/installed successfully.

**Comment by alfredh** [comment]
looks like the module is included twice in the build.

how is the makefile invoked ?

you can try this:

```
$ make clean
$ make STATIC=yes
```

can you also provide the full build log and the output of:

```
$ make info
$ make
```


Alfred

**Comment by alfredh** [comment]
looks like the omx module is added twice if both RPI and Bellagio
headers are present:

```
ifneq ($(USE_OMX_RPI),)
MODULES   += omx
endif
ifneq ($(USE_OMX_BELLAGIO),)
MODULES   += omx
endif
```

which on takes precedence ?

could you try to comment out the 3 lines for bellagio and build again ?

**Comment by lisleadam** [comment]
thanks alfredh.  you're right. commenting out the bellagio lines in mk/modules.mk surely did the trick.

**Comment by alfredh** [comment]
hi @lisleadam 

many thanks for reporting !

fixed in commit e43d303c910561112206a61a4da706c03432cf3f
